js/rpg_windows/Window_MenuStatus.js

```diff
@@ -43,7 +43,7 @@ Window_MenuStatus.prototype.numVisibleRows = function() {
 
 Window_MenuStatus.prototype.loadImages = function() {
     $gameParty.members().forEach(function(actor) {
-        ImageManager.loadFace(actor.faceName());
+        ImageManager.reserveFace(actor.faceName());
```

Comment:
Could you add `reserveFace` also in `Window_Status` and `Window_SkillStatus`?
They use `drawFace`, so it was necessary in the first place, but it was not written.